packages/boxed-expression-component/src/expressions/FunctionExpression/FunctionKindSelector.tsx

```diff
@@ -45,6 +44,73 @@ export const FunctionKindSelector: React.FunctionComponent<FunctionKindSelectorP
     [onFunctionKindSelect]
   );
 
+  function iconForFunctionKind(functionKind: FunctionExpressionDefinitionKind): {
+    icon: React.ReactNode;
+    description: String;
+  } {
+    switch (functionKind) {
+      case FunctionExpressionDefinitionKind.Feel:
+        return {
+          icon: (
+            <span
+              style={{
+                fontSize: "0.8em",
+                fontWeight: "bold",
+              }}
+            >
+              FEEL
+            </span>
+          ),
+          description:
+            "Define function as a 'Friendly Enough Expression Language (FEEL)' expression. This is the default.",
+        };
+      case FunctionExpressionDefinitionKind.Java:
+        return {
+          icon: (
+            <span
+              style={{
+                fontSize: "0.8em",
+                fontWeight: "bold",
+              }}
+            >
+              JAVA
+            </span>
+          ),
+          description:
+            "Define full qualified java class name and a public static method signature to invoke.\nThe method-signature consist of the name of the method, followed by an argument list of the argument types.",
```

Comment:
@jomarko Some minor fixes to the text:
```suggestion
            "Define the full qualified java class name and a public static method signature to invoke.\nThe method signature consists of the name of the method, followed by an argument list of the argument types.",
```

It would be GREAT to put an example here too, like:

**Class name**: `org.kie.kogito.MyClass`
**Method signature**: `doSomething(Integer, String, String)`

I'm not sure if the example above is right, though :P